src/interface/web/next.config.mjs

```diff
@@ -0,0 +1,46 @@
+/** @type {import('next').NextConfig} */
+
+const isProd = process.env.NEXT_PUBLIC_ENV === 'production';
+
+const nextConfig = {
+    output: isProd ? 'export' : undefined,
+    rewrites: isProd ? undefined : async () => {
+        return [
+            {
+                source: '/api/:path*',
+                destination: 'http://localhost:42110/api/:path*',
+            },
+        ];
+    },
+    trailingSlash: true,
+    skipTrailingSlashRedirect: true,
+    distDir: 'out',
+    images: {
+        loader: isProd ? 'custom' : 'default',
+        loaderFile: isProd ? './image-loader.ts' : undefined,
+        remotePatterns: isProd ? [
+            {
+                protocol: "https",
+                hostname: "**.googleusercontent.com",
+            },
+            {
+                protocol: "https",
+                hostname: "generated.khoj.dev",
+            },
+            {
+                protocol: "https",
+                hostname: "assets.khoj.dev",
+            },
+            {
+                protocol: "https",
+                hostname: "khoj-web-bucket.s3.amazonaws.com",
+            },
+            {
+                protocol: "https",
+                hostname: "khoj-generated-images.s3.amazonaws.com",
+            }
```

Comment:
These two s3 buckets shouldn't be required anymore? Those buckets are deprecated